locale/menu.xml

```diff
@@ -24,7 +24,7 @@
       </acls>
     </menu-item>
   </menu-item>
-  <menu-item id="1" label="AR">
+  <menu-item id="1" label="Accounts Receivable">
```

Comment:
Do we have a way to update the translations, so that existing translations of "AR" will instead match "Accounts Receivable"? Or are we just going to accept breakage, then have a push for translations at a later date?

Reply:
At this point, I'm happy to accept the breakage because this should be 3 minutes of work for all translators and it's apparently more work on my side...

Reply:
Solved. I've backported these with the help of Text::PO::Parser :tada: